Makefile

```diff
@@ -1,3 +1,10 @@
+SHELL := /usr/bin/env bash
+OUT_DIR = dist
+TEMPLATE_FILE=template.yml
+CF_BUCKET=cloudformation-`aws sts get-caller-identity --output text --query 'Account'`-$(AWS_DEFAULT_REGION)
```

Comment:
AFAIK `$(AWS_DEFAULT_REGION)` will not resolve unless you add this line above `AWS_DEFAULT_REGION ?= $(shell aws configure get region)`